docs/src/main/sphinx/release/release-434.md

```diff
@@ -0,0 +1,102 @@
+# Release 434 (20 Nov 2023)
+
+## General
+
+* {{breaking}} Rename the `query.max-writer-tasks-count` configuration property
+  and the related session property `max_writer_tasks_count` to
+  `query.max-writer-task-count` and `max_writer_task_count`. ({issue}`19793`)
+
+## Security
+
+## Web UI
+
+## JDBC driver
+
+## Server RPM
+
+## Docker image
+
+## CLI
+
+## Accumulo connector
+
+## Atop connector
+
+## BigQuery connector
+
+* Support reading `json` type. ({issue}`19790`)
+
+## Blackhole connector
+
+## Cassandra connector
+
+## ClickHouse connector
+
+## Delta Lake connector
+
+* Improve performance when reading large checkpoint files on partitioned tables.
```

Comment:
Should probably include a link to documentation on what checkpoint files are.

Reply:
not sure where to link to - seems like we used https://docs.delta.io/latest/delta-batch.html#-data-retention .. but thats kinda odd

Reply:
and we didnt link in some other release notes either ..